Blizzard.IO.RabbitMQ.Tests/Rpc/RpcRabbitWrapperTests.cs

```diff
@@ -0,0 +1,107 @@
+﻿using Blizzard.IO.RabbitMQ.Rpc;
+using Blizzard.IO.RabbitMQ.Entities.Rpc;
+using NUnit.Framework;
+using EasyNetQ;
+using Moq;
+using EasyNetQ.Producer;
+using System;
+using EasyNetQ.Topology;
+using System.Threading.Tasks;
+
+namespace Blizzard.IO.RabbitMQ.Tests.Rpc
+{
+    class CncelationMock : IDisposable
+    {
+        public void Dispose()
+        {}
+    }
+
+    class ConventionStub : IConventions
+    {
+        public ExchangeNameConvention ExchangeNamingConvention { get; set; }
+        public TopicNameConvention TopicNamingConvention { get; set; }
+        public QueueNameConvention QueueNamingConvention { get; set; }
+        public RpcRoutingKeyNamingConvention RpcRoutingKeyNamingConvention { get; set; }
+        public ErrorQueueNameConvention ErrorQueueNamingConvention { get; set; }
+        public ErrorExchangeNameConvention ErrorExchangeNamingConvention { get; set; }
+        public RpcExchangeNameConvention RpcRequestExchangeNamingConvention { get; set; }
+        public RpcExchangeNameConvention RpcResponseExchangeNamingConvention { get; set; }
+        public RpcReturnQueueNamingConvention RpcReturnQueueNamingConvention { get; set; }
+        public ConsumerTagConvention ConsumerTagConvention { get; set; }
+    }
+
+    [TestFixture]
+    public class RpcRabbitWrapperTests
+    {
+        private RpcRabbitWrapper _rpcRabbitWrapper;
+        private RpcConfiguration _rpcConfiguration;
+
+        private Mock<IAdvancedBus> _advancedBusMock;
+        private Mock<IEventBus> _eventBusMock;
+        private IConventions _conventionsStub;
```

Comment:
Change the type here to `ConventionStub` there's no need to keep it as an interface in the tests,
you want it to be `ConventionStub` and no other implementation

Reply:
f